pages/common/acme.sh.md

```diff
@@ -0,0 +1,36 @@
+# acme.sh
+
+> An ACME Shell script, an acme client alternative to certbot.
+> More information: <https://github.com/acmesh-official/acme.sh>
+
+- Issue a certificate using webroot mode:
+
+`acme.sh --issue --domain {{domain.tld}} --webroot {{/path/to/webroot}}`
+
+- Issue a certificate using standalone mode using port 80:
+
+`acme.sh --issue --standalone --domain {{domain.tld}}`
+
+- Issue a certificate using standalone TLS mode using port 443:
+
+`acme.sh --issue --alpn --domain {{domain.tld}}`
+
+- Issue a certificate using a working Nginx configuration:
+
+`acme.sh --issue --nginx --domain {{domain.tld}}`
+
+- Issue a certificate using a working Apache configuration:
+
+`acme.sh --issue --apache --domain {{domain.tld}}`
+
+- Issue a wildcard (\*) certificate using a manual DNS mode:
+
+`acme.sh --issue --dns --domain {{*.domain.tld}}`
+
+- Issue a certificate using an automatic DNS API mode:
+
+`acme.sh --issue --dns {{dns_api}} --domain {{domain.tld}}`
+
+- Install certificate file into location specified by its parameter (not displayed):
+
+`acme.sh --install-cert --domain {{domain.tld}} --reloadcmd {{reload_commad}}`
```

Comment:
This would ideally contain a far longer code snippet, like:

```
acme.sh --install-cert -d {{domain.tld}}  \
  --key-file {{/etc/nginx/ssl/domain.tld.key}} \
  --fullchain-file {{/etc/nginx/ssl/domain.tld.cer}} \
  --reloadcmd {{"systemctl force-reload nginx"}}
```

But I am not sure how to proceed. All the parameters, except the `--domain` are optional. However, there are many parameters in a format `--*-file`handling their respective certificate key files. Without a single *-file parameter, the whole command does literally nothing.

Additionally, `--reloadcmd` is also optional, but it is run without it only rarely (in the situations that the service restart is done completely manually, bypassing the whole point of automatic certificate renewal mechanism).

I am not entirely sure how to proceed, but this single code block is the only one I would've looked for when calling tldr acme.sh.

Reply:
@bl-ue @marchersimon @sbrl please, can anyone provide any kind of feedback?

Should this stay as it is, or is there any way to display long command? I did not find anything "long" related in the contribution guide.

Reply:
This seems a little long, but it might not be... 🤔 
```markdown
`acme.sh --install-cert -d {{example.com}} --key-file {{/path/to/example.com.key}} --fullchain-file {{/path/to/example.com.cer}} --reloadcmd {{"systemctl force-reload nginx"}}`
```

Reply:
I also think we should try to stuff it into one line. Mainly because most clients don't support multiline code.

Reply:
Guys please, I am not sure what to do here. Would appreciate some guidance, if I am expected to commit some changes.

Reply:
4. add the subcommand-like page, detailing the most used `--install-cert` options, as there are quite a few of them and mention this page in See also section

Reply:
Why not just stuff it into one line?  It's a little long, but I don't really see any strong reason why not to.  As for multiline, we can't do that, because that goes against the client spec (or at least the client spec will have to be updated).

Reply:
I support any idea of documenting the `--install-cert` option, as it has around 5-7 commonly used parameters.

Thinking about it further, the name of the file could probably be called `acme.sh-install-cert.md` similarly as for instance `git-cat-file.md`

Would this somehow break existing conventions?

Reply:
For some commands, it's unavoidable to have long commands - just do `tldr whiptail` and you'll see what I mean. It's not necessarily a _bad_ thing. I say put it all on 1 line.